SDKs/python_sdk/import_products_from_shopify_csv/README.md

```diff
@@ -0,0 +1,29 @@
+# Import products from Shopify CSV file
+
+This example demonstrates how to import a bulk of product from a CSV file.
+It uses:
+- The **[builton-sdk](https://pypi.org/project/builton-sdk/)**
+- A **Service Account** Key (SA Key), you can find or create one through [BuiltOn](http://dashboard.builton.dev/) dashboard section **Settings**, sub-section **Service Account Keys**.
+- A **BuiltOn API** Key, you can find or create one through [BuiltOn](http://dashboard.builton.dev/) dashboard section *Settings*, sub- section **API Keys**.
+
+# Install Locally
+
+In order to run this example locally, you need to:
+
+1. Clone this repository, and run `pip install -r requirements.txt`
+
+    1.1. You can also create a Virtual Environment: `python3 -m venv venv`
```

Comment:
I'd suggest to create the virtual environment (`python3 -m venv venv`) before the `pip install`